### DISCUSSION TITLE
Session 8 exercise

### DISCUSSION THREAD

**Discussion by Irenekyei1** [LiNk-NY/IntroR#46] [General]
I want to ask for clarity on questions 1 and 2 in the assignment. Should the answers be written in code chunks or just like a text/comment? I didn't catch it properly when @ralythompson24  explained it in class last night.

**Comment by ewhittemore987** [comment]
Good afternoon @Irenekyei1 

The answers for 1 and 2 are written simply as text (with the proper R Markdown syntax to produce your desired result). 

This R Markdown cheat sheet helped me during last semester in Biostatistics. All of our assignments were turned in as PDFs knitted from .rmd files, so having the right syntax was important. I found this document very helpful to understand the R Markdown syntax (language). Hope this helps!

[R Markdown Cheatsheet](https://www.rstudio.com/wp-content/uploads/2015/02/rmarkdown-cheatsheet.pdf)

**Reply by Irenekyei1** [reply]
Thank you so much for your response and the cheat sheet @ewhittemore987. They are really helpful
